build_runner/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.1.1
+
+### Bug Fixes
+
+- Handle re-snapshotting the build script on SDK updates.
+- Add `Bootstrap` to the list of known loggers.
```

Comment:
"known loggers" is an implementation detail - I'd maybe  go with "Suppress header for the `Bootstrap` logger."